src/face_tools.rs

```diff
@@ -0,0 +1,95 @@
+use nalgebra::base::Vector3;
+
+use crate::convert::cast_usize;
+use crate::geometry::{Geometry, TriangleFace};
+
+#[allow(dead_code)]
+pub fn calculate_triangle_face_normal(
```

Comment:
These tests are useful, already written and aren't interfering with anything. I don't get how no tests would be more benefitial.

Reply:
I've removed the redundant function, however I kept the already existing tests - mainly because I wanted to make sure the replacement function works exactly the same like the removed one.